blueprints/ember-bootstrap/index.js

```diff
@@ -178,13 +180,24 @@ module.exports = {
     let file = 'ember-cli-build.js';
     let bootstrapVersion = this.bootstrapVersion;
     let preprocessor = this.preprocessor;
+    let config = this.retrieveExistingConfiguration();
     let settings = {
       bootstrapVersion
     };
 
-    settings.importBootstrapFont = (bootstrapVersion === 3);
+    if (bootstrapVersion === 4) {
+      settings.importBootstrapFont = false;
+    } else if (config.hasOwnProperty('importBootstrapFont')) {
+      settings.importBootstrapFont = config.importBootstrapFont;
+    } else {
+      settings.importBootstrapFont = true;
+    }
 
-    settings.importBootstrapCSS = (preprocessor === 'none');
+    if (config.hasOwnProperty('importBootstrapCSS')) {
+      settings.importBootstrapFont = config.importBootstrapFont;
```

Comment:
It passed because the config editor doesn't change or remove properties that aren't in the config being set.